src/CabanaPD_BodyTerm.hpp

```diff
@@ -34,12 +38,17 @@ struct BodyTerm
     template <class ExecSpace, class ParticleType>
     void apply( ExecSpace, ParticleType& particles, const double time )
     {
+        _timer.start();
         Kokkos::RangePolicy<ExecSpace> policy( 0, particles.n_local );
         auto user = _user_functor;
         Kokkos::parallel_for(
             "CabanaPD::BodyTerm::apply", policy,
             KOKKOS_LAMBDA( const int p ) { user( p, time ); } );
+        _timer.stop();
     }
+
+    auto time() { return _timer.time(); };
```

Comment:
Why semicolon goes inside curly brackets in "{ return _timer.time(); }" ?

Reply:
End of the line; it just happens to be a one line function